applications/CompressiblePotentialFlowApplication/custom_strategies/potential_flow_residualbased_incrementalupdate_static_scheme.h

```diff
@@ -115,49 +115,32 @@ class PotentialFlowResidualBasedIncrementalUpdateStaticScheme : public ResidualB
     }
 
     /**
-     * @brief Default constructor. The pseudo static scheme (parameters)
-     * @param ThisParameters Dummy parameters
+     * @brief Default constructor.
+     * @param ThisParameters See parameters
      */
     explicit PotentialFlowResidualBasedIncrementalUpdateStaticScheme(Parameters ThisParameters) : BaseType()
     {
         // Validate and assign defaults
         ThisParameters = this->ValidateAndAssignParameters(ThisParameters, this->GetDefaultParameters());
         this->AssignSettings(ThisParameters);
-    }
+}
 
     /**
      * @brief Default constructor.
-     * @param ElementType Type of element used in the simulation.
-     * @param InitialCriticalMach Initial critical mach value.
-     * @param InitialUpwindFactorConstant Initial upwind factor constant value.
-     * @param TargetCriticalMach Critical mach value to be used once the UpdateRelativeResidualNorm value is reached.
-     * @param TargetUpwindFactorConstant Upwind factor constant value to be used once the UpdateRelativeResidualNorm value is reached.
-     * @param UpdateRelativeResidualNorm Relative residual norm value to which the critical mach and the upwind factor constant will be updated.
-     * @param MachNumberSquaredLimit Mach number squared limit value wich will be used.
+     * @param ThisParameters See parameters
      */
-    explicit PotentialFlowResidualBasedIncrementalUpdateStaticScheme(
-        std::string ElementType,
-        double InitialCriticalMach,
-        double InitialUpwindFactorConstant,
-        double TargetCriticalMach,
-        double TargetUpwindFactorConstant,
-        double UpdateRelativeResidualNorm,
-        double MachNumberSquaredLimit) 
-        : BaseType()
+    explicit PotentialFlowResidualBasedIncrementalUpdateStaticScheme(Parameters ThisParameters, bool IsTransonic) : BaseType()
```

Comment:
Why don't we also pass the `IsTransonic` by the parameters?

Reply:
I don't know, haha